Layman-eUTxO-Design-Course/batch-1/teams/ergo-school/ergo-school-summary.md

```diff
@@ -0,0 +1,14 @@
+# Ergo School
```

Comment:
This file and folder can be deleted. Only keep the one in team. (I may have said teams in the past, my bad)